docs/ml-classification-regression.md

```diff
@@ -374,6 +374,197 @@ regression model and extracting model summary statistics.
 
 </div>
 
+## Generalized linear regression
+
+When working with data that has a relatively small number of features (< 4096), Spark's GeneralizedLinearRegression interface
+allows for flexible specification of [generalized linear models](https://en.wikipedia.org/wiki/Generalized_linear_model) (GLMs) which can be used for various types of
+prediction problems including linear regression, Poisson regression, logistic regression, and others.
```

Comment:
I think it's better to document which are Spark's GeneralizedLinearRegression supported.